doc/man7/provider-encoder.pod

```diff
@@ -152,6 +152,14 @@ An implementation with that output type outputs PEM formatted data.
 
 An implementation with that output type outputs DER formatted data.
 
+=item msblob
```

Comment:
Not sure that it matters but these items are all uppercase in the code.

Reply:
Not really. The properties are specified as macro parameters in encoders.inc as `ENCODER("RSA", rsa, yes, msblob),` As you can see the msblob is lowercase.